example/package.json

```diff
@@ -8,6 +8,7 @@
     "build:android": "yarn build:_pre && react-native bundle --entry-file index.js --platform android --dev true --bundle-output dist/main.android.jsbundle --assets-dest dist/res",
     "build:ios": "yarn build:_pre && react-native bundle --entry-file index.js --platform ios --dev true --bundle-output dist/main.ios.jsbundle --assets-dest dist",
     "build:macos": "yarn build:_pre && react-native bundle --entry-file index.js --platform macos --dev true --bundle-output dist/main.macos.jsbundle --assets-dest dist",
+    "build:visionos": "yarn build:_pre && react-native bundle --entry-file index.js --platform ios --dev true --bundle-output dist/main.visionos.jsbundle --assets-dest dist",
```

Comment:
Setting to build platform iOS. for now, Need to update either RNM or rex-kit to recognize visionOS here.